code/modules/mob/living/carbon/alien/adult/queen.dm

```diff
@@ -181,3 +181,15 @@
 	if(!silent)
 		to_chat(user, span_noticealien("You discard [src]."))
 	return ..()
+
+/mob/living/carbon/alien/adult/royal/queen/attackby(obj/item/O, mob/user, params)
+	if(istype(O, /obj/item/clothing/under/costume/maid) || istype(O, /obj/item/clothing/under/rank/civilian/janitor/maid))
+		if(do_after(user, 5 SECONDS, src))
+			maidify()
+
+/mob/living/carbon/alien/adult/royal/queen/proc/maidify()
+	name = "alien queen maid"
+	desc = "Lusty, Sexy"
+	icon_state = "alienqmaid"
```

Comment:
shouldn't this be in update_icon_state/update_name/update_desc alongside a bool on the queen?

Reply:
Name updates will add the others.

Reply:
sike idk how to do this

Reply:
want to talk on discord?

Reply:
Sure, I'll send a message.

Reply:
I did something, it seems to work. Hope it works for you too.